### PR TITLE
battleship WIP

### PR DESCRIPTION
**Pull request by matsepiol**

### CODE REVIEW

File: js/computerAI.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+var computerAI = (function() {
+  var columnArray = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J'],
+      shotArray = [];
+
+  var helper = window.helper;
```
Comment (rootical) [inline js/computerAI.js:7]: Why not passing it inside instead of using window?

Comment (matsepiol) [inline js/computerAI.js:7]: You mean passing it as a function parameter? Yes, could be done this way as well.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,867 @@
...
+
+
+    <div class="confirmation hidden">
+      <span></span>
+      <button class="button" type="button" onclick="confirmShip();">Confirm</button>
+    </div>
+
+    <div class="grids">
+      <div id="user-grid">
+        <div class="grid-element grid-letter grid-num" id="0-0">
```
Comment (rootical) [inline index.html:35]: Would be nice if you would try to generate it. 
You could use any template engine for this supporting looping. Or just generate html yourself.

**Comment by rootical** [comment]
You may add improvements in other PRs. Great Job!

**Pull request merged**